### ISSUE TITLE
Add kenney game assets

### ISSUE THREAD

**Issue by Stedders** [Stedders/coder-dojo-common-python#2]
Include several assets https://kenney.nl/assets in the package to allow attendees to get up and running with the examples as quickly as possible.

**Comment by dannystaple** [comment]
Thoughts on this - perhaps this could be a script, depending on requests (and maybe a zip package) to download and install. Also, as Jonathan suggested, it could be installable offline from physical media, perhaps using a command line flag.

**Comment by Stedders** [comment]
Yeah, I was going to build some tooling around it to download the packages.

The offline comment is a good point. I'll put something together for review.

**Comment by Stedders** [comment]
I've been implementing something along those lines...

```json
{
    "assets": [
        {
            "archive": null,
            "local": null,
            "name": "tiny-town",
            "url": "https://kenney.nl/assets/tiny-town"
        },
        {
            "archive": null,
            "local": null,
            "name": "pattern-pack-pixel",
            "url": "https://kenney.nl/assets/pattern-pack-pixel"
        },
        ...
```
I hope to have something by Saturday, and we can discuss it.

**Comment by dannystaple** [comment]
That sounds awesome. Looking at the list, I guess we are talking about 3 modes of operation? 
- install from the internet
- Download from the internet into a local folder (to make a USB stick)
- install from the local folder

**Comment by Stedders** [comment]
I'm making it flexible enough to allow these options.

The MVP will install from the Internet, and we can then extend it.

**Comment by dannystaple** [comment]
100% behind making an internet install mvp. Perhaps a couple of tickets to break down the other options then?Sent from my iPhoneOn 21 Jan 2023, at 09:00, Stedders ***@***.***> wrote:﻿
I'm making it flexible enough to allow these options.
The MVP will install from the Internet, and we can then extend it.
